src/types/Comment.ts

```diff
@@ -0,0 +1,7 @@
+export interface Comment {
```

Comment:
Add a space after the opening curly brace and before the closing curly brace of the Comment interface. For example, `export interface Comment { ... }`